lib/generators/rspec/generator/generator_generator.rb

```diff
@@ -3,8 +3,8 @@
 module Rspec
   module Generators
     # @private
-    class GeneratorsGenerator < Base
-      class_option :generator_specs, :type => :boolean, :default => false,  :desc => "Generate generator specs"
+    class GeneratorGenerator < Base
+      class_option :generator_specs, :type => :boolean, :default => true,  :desc => "Generate generator specs"
```

Comment:
I'm not sure we want to flip the default state here, generally speaking generators are a rarely used feature so I think its fine to have to request them.

Reply:
I'm not strongly against it, however, this may be quite confusing if we rely on the rails documentation [[1]](https://guides.rubyonrails.org/generators.html#customizing-your-workflow)

The following command will silently ignore the spec 

```
bin/rails generate generator my_awesome_generator
```

We need to explicitly ask for 

```
bin/rails generate generator my_awesome_generator --generator-specs
```

Which it sounds a bit unnatural to me, but your call here.

Reply:
I freely admit I might be misunderstanding the hierarchy here. I agree that:

```
bin/rails generate generator my_generator
```

should work, but in no other circumstance should you end up with them without adding `--generator-specs`

Reply:
`bin/rails generate generator my_generator` works.

It just a matter of default behavior here.

No Spec generated:
```
bin/rails generate generator foo
       exist  lib/generators/foo
      create  lib/generators/foo/foo_generator.rb
      create  lib/generators/foo/USAGE
       exist  lib/generators/foo/templates
      invoke  rspec
```

Spec generated:
```
bin/rails generate generator bar --generator-specs
       exist  lib/generators/bar
      create  lib/generators/bar/bar_generator.rb
      create  lib/generators/bar/USAGE
       exist  lib/generators/bar/templates
      invoke  rspec
      create    spec/generator/bars_generator_spec.rb
```

Your call.